src/js/video.js

```diff
@@ -140,8 +140,8 @@ videojs.hooks_ = {};
  * @param {string} type
  *        the lifecyle to get hooks from
  *
- * @param {Function} [fn]
- *        Optionally add a hook to the lifecycle that your are getting.
+ * @param {Function|Function[]} [fn]
```

Comment:
I don't think we should document or officially support multiple handlers in a single hooks call. It just encourages bad code.

Reply:
It's already documented for the other function. Was trying to be consistent.

Reply:
So it is. Carry on.